pkg/generator/deploy_tmpl.go

```diff
@@ -33,19 +33,23 @@ const operatorYamlTmpl = `apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{.ProjectName}}
+  labels:
+    k8s-app: {{.ProjectName}}
 spec:
   replicas: 1
   selector:
     matchLabels:
-      name: {{.ProjectName}}
+      k8s-app: {{.ProjectName}}
```

Comment:
why change `name` to `k8s-app`?

Reply:
I introduced `k8s-app` label for service and servicemonitor. I just changed to have the same label everywhere.

If needed I can change to use `name` label in service and servicemonitor

Reply:
I think we should keep the the original label `name` for now. if change is indeed needed, we can change the label to something else in a future pr.

cc/ @spahl @hasbro17 on label change.

Reply:
Okay for using `name` label.
Should I use `name` label in `service` and `servicemonitor` too ?